cypress/integration/product/change_product_price_spec.js

```diff
@@ -0,0 +1,90 @@
+import { Product, ProductCategory, ProductPrice} from '../../support/utils/product';
+import { SalesOrder, SalesOrderLine} from '../../support/utils/sales_order';
+import { BPartner} from '../../support/utils/bpartner';
+import { BPartnerLocation } from '../../support/utils/bpartner_ui';
+
+describe('Create Product', function() {
+  const timestamp = new Date().getTime(); 
+  const productName = `ProductName ${timestamp}`;
+  const productValue = `ProductNameValue ${timestamp}`;
+  const productCategoryName = `ProductCategoryName ${timestamp}`;
+  const productCategoryValue = `ProductNameValue ${timestamp}`;
+  const bpName = `Customer ${timestamp}`;
+  let salesOrderRecordID;
+  let productRecordID;
+
+  it('Create a new ProductCategory', function() {
+    cy.fixture('product/simple_productCategory.json').then(productCategoryJson => {
+      Object.assign(new ProductCategory(), productCategoryJson)
+        .setName(productCategoryName)
+        .setValue(productCategoryValue)
+        .apply();
+    });
+  });
+
+  it('Create a Product and Price', function() {
+    const productPrice = new ProductPrice()
+      .setPriceList("Testpreise Kunden (Deutschland)_Germany - Deutschland_EUR_2015-01-01")
+      .setTaxCategory("Regular Tax Rate 19% (Germany)")
```

Comment:
PriceList and TaxCategory should be extracted to their own constants. Too much _magic_ in the code.

Reply:
What do you mean by this?

Reply:
I mean that at some point we might want to change the PriceList from `Testpreise Kunden` to some pricelist created specifically for our test. In this case having all the constants at the top would help us see what needs to be created/edited a little bit easier.

Reply:
Oh, ok, sure 👍